utils/banks.js

```diff
@@ -161,6 +170,9 @@ export async function getBankDetail (bankTag) {
           website,
           commentary ${commentaryFields},
           bankFeatures ${bankFeaturesFields}
+          countries {
```

Comment:
need this so I can determine which country a specific bank page is so that we show the breakup letter only on UK pages